integration/cli/build.configequality.test.ts

```diff
@@ -0,0 +1,424 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023-Present The Pepr Authors
+
+import { beforeAll, describe, expect, it } from "@jest/globals";
+import * as path from "node:path";
+import * as fs from "node:fs/promises";
+import { kind } from "kubernetes-fluent-client";
+import { Workdir } from "../helpers/workdir";
+import * as time from "../helpers/time";
+import * as pepr from "../helpers/pepr";
+import * as resource from "../helpers/resource";
+import { ModuleConfig } from "../../src/lib/core/module";
+
+const FILE = path.basename(__filename);
+const HERE = __dirname;
+
+describe("build", () => {
+  const workdir = new Workdir(`${FILE}`, `${HERE}/../testroot/cli`);
+
+  beforeAll(async () => {
+    await workdir.recreate();
+  }, time.toMs("1m"));
+
+  describe("builds a module", () => {
+    const id = FILE.split(".").at(1);
+    const testModule = `${workdir.path()}/${id}`;
+    const packageJson = `${testModule}/package.json`;
+
+    beforeAll(async () => {
+      await fs.rm(testModule, { recursive: true, force: true });
+      const argz = [
+        `--name ${id}`,
+        `--description ${id}`,
+        `--errorBehavior reject`,
+        "--confirm",
+        "--skip-post-init",
+      ].join(" ");
+      await pepr.cli(workdir.path(), { cmd: `pepr init ${argz}` });
+      await pepr.tgzifyModule(testModule);
+      await pepr.cli(testModule, { cmd: `npm install` });
+    }, time.toMs("2m"));
+
+    describe("maps config options from package.json into pepr + helm manifests", () => {
+      let moduleConfig: Required<ModuleConfig>;
+      let peprResources: any; // eslint-disable-line @typescript-eslint/no-explicit-any
+      let helmResources: any; // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
Can we avoid `any` and specify a type instead? If not, does that warrant an issue to add better typing later?

Reply:
Nope, those are _supposed_ to be `any` values since they hold random, non-typed values as read-in directly from .yaml files.

Before we use anything out of those variables for testing though, we're sub-selecting items from within them -- using resource.select() calls a bit further down -- and doing _that_ is strongly typing them (so they're not random any's by the time we assert on them.